### PR TITLE
Eduardo jimenez

### PR DESCRIPTION
**Pull request by l8nightswithJS**
Sprint complete all commits finalized and pushed.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -11,58 +11,67 @@
...
+
+    <article class="bottom-about">
+      <h2>Let's Work Together</h2>
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+    </article>
+      <footer class="nav-about">
         <nav>
           <a href="index.html">Home</a>
-          <a href="#">About</a>
+          <a href="C:\Users\ejmrt\LambdaCode\Sprint-Challenge--User-Interface\about.html">About</a>
```
Comment (MisterC-Rogers) [inline about.html:74]: this should look like the one above it

File: index.html [hunk 2]
```diff
@@ -12,6 +12,19 @@
 </head>
 
 <body>
+    <nav class="nav-container">
+            <a href="C:\Users\ejmrt\LambdaCode\Sprint-Challenge--User-Interface\index.html"><img class="logo" src="img/lambda-black.png" alt="Great Idea! Company logo."></a>
+            <a href="C:\Users\ejmrt\LambdaCode\Sprint-Challenge--User-Interface\index.html">Home</a>
```
Comment (MisterC-Rogers) [inline index.html:17]: when call the page the [href = 'index.html'] is just fine

**Review by MisterC-Rogers — COMMENTED** [review]
things look good other then the comments I added

File: about.html (hunk 1 above)
Comment (l8nightswithJS) [reply about.html:74]: Thanks will definitely remember that.

**Pull request merged**